lib/mrn_value_decoder.cpp

```diff
@@ -17,9 +17,10 @@
   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
+#include <mrn_mysql_compat.h>
```

Comment:
Including after `mrn_value_decoder.hpp` is better to not affect definitions in `mrn_value_decoder.hpp`.

Reply:
I got it.